snap/snapcraft.yaml

```
@@ -72,6 +72,14 @@ environment:
 
   KNN_LIB_DIR: ${OPENSEARCH_PLUGINS}/opensearch-knn/lib
 
+
+slots:
+  logs:
+    interface: content
+    source:
+      read:
+        - $SNAP_COMMON/ops/snap/logs
```

all DP snaps must follow our directory structure standard https://docs.google.com/document/d/1MyT4Wt_-vgPwhwEjC-oeHRdLYrZWaQzFGTwclT7hD1U/edit 
```suggestion
        - $SNAP_COMMON/var/log/kafka
```

---

Fixed.